cdc/entry/mounter.go

```diff
@@ -386,53 +388,79 @@ func (m *mounterImpl) mountRowKVEntry(tableInfo *TableInfo, row *rowKVEntry) (*m
 			whereHandle := true
 			col.WhereHandle = &whereHandle
 		}
-		values[colName] = col
+		cols[colName] = col
+	}
+	for _, col := range tableInfo.Columns {
+		_, ok := cols[col.Name.O]
+		if !ok && tableInfo.IsColWritable(col) {
+			column := &model.Column{
+				Type:  col.Tp,
+				Value: getDefaultOrZeroValue(col),
+			}
+			if tableInfo.IsColumnUnique(col.ID) {
+				whereHandle := true
+				column.WhereHandle = &whereHandle
+			}
+			cols[col.Name.O] = column
+		}
+	}
+	return cols, nil
+}
+
+func (m *mounterImpl) mountRowKVEntry(tableInfo *TableInfo, row *rowKVEntry) (*model.RowChangedEvent, error) {
+	if !m.enableOldValue && row.Delete && !tableInfo.PKIsHandle {
+		return nil, nil
+	}
+
+	var err error
+	// Decode changed columns.
+	var changedCols map[string]*model.Column
+	if len(row.ChangedRow) != 0 {
+		changedCols, err = datum2Column(tableInfo, row.ChangedRow)
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+	}
+	// Try to decode columns, if row represents a delete event, columns should be == changed columns
+	var cols map[string]*model.Column
+	if !row.Delete {
```

Comment:
if the `enableOldValue` is `false` and `PKIsHandle` is `true`, it seems like we lost the handle column value here